src/Components/Card/index.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import { AiFillEdit, AiOutlineCheck, AiOutlineClose } from "react-icons/ai";
+import './style.css';
+
+const Card = (props) => {
+  // Card style switcher
+  let [chooseСardFlag, setChooseСardState] = useState(false);
+  const selectItem = () => setChooseСardState(!chooseСardFlag);
+  const clsStyle = chooseСardFlag === false ? 'card_is_green' : 'card_is_red';
+
+  // Switch between edit mode and normal mode
+  let [editModeFlag, setCardState] = useState(false);
+  const changeText = () => {
+    setChooseСardState(chooseСardFlag = false);
+    setCardState(!editModeFlag);
+  };
+
+  // Adding new text to cards
+  const [textData, setNewTextData] = useState({
+    ...props.card
+  });
+  const addNewText = event => {
+    textData[event.target.name] = event.target.value;
+    setNewTextData(textData);
```

Comment:
Looks a bit odd to me. You're first directly mutating the state, then re-assign it with the same values? This is bad practice. You can just call setNewTextData with object copy by using spread operator:
```
const addNewText = event => {
  setNewTextData({
    ...textData,
    [event.target.name] = event.target.value,
    });
}
```

Reply:
Yes I agree with you
Fixed